src/routes/messages/entities/created-message.entity.ts

```diff
@@ -0,0 +1,23 @@
+import { ApiProperty, ApiPropertyOptional } from '@nestjs/swagger';
+import { CreatedMessageConfirmation } from './created-message-confirmation.entity';
+
+export class CreatedMessage {
```

Comment:
This is equivalent to the domain `Message` entity, and different from the route `Message` entity. If we want to return a route entity in the controller we would need a class like this (to avoid adding OpenAPI decorators to the domain entity). 

Another option is to just return `Promise<unknown>` and pass the object coming from the transaction service, just as we do in [DelegatesController](https://github.com/5afe/safe-client-gateway-nest/blob/7cf4032e7d442c37b2843341aeaf2491df321713/src/routes/delegates/delegates.controller.ts#L82) for example when creating/deleting a delegate. This sounds like the way to go, what do you think?

Reply:
I've removed `CreatedMessage`/`CreatedMessageConfirmation` entities and changed the return type to `Promise<unknown>` in [5aa4d27](https://github.com/5afe/safe-client-gateway-nest/pull/281/commits/5aa4d2702f1b0fb36dd1f5dda2450febcb4cbb69)